src/Examples/UnitModels/Operations/feed_testing.ipynb

```diff
@@ -0,0 +1,376 @@
+    "\n",
+    "\n",
+    "![](feed.svg)\n",
+    "![feed.svg](attachment:feed.svg)\n",
+    "\n",
+    "## Learning Outcomes\n",
+    "\n",
+    "- Demonstrate use of the Feed unit model in IDAES\n",
+    "\n",
+    "\n",
+    "## Problem Statement\n",
+    "\n",
+    "In this tutorial, we will be feeding a mixed stream of liquid benzene and liquid toluene. This unit model has no inlets and only one outlet. The outlet conditions are as follows:\n",
+    "\n",
+    "Flow Rate = 100 mol/s\n",
+    "\n",
+    "Mole Fraction (Benzene) = 0.6\n",
+    "\n",
+    "Mole Fraction (Toluene) = 0.4\n",
+    "\n",
+    "Temperature = 298.15 K\n",
+    "\n",
+    "Pressure = 101325 Pa\n",
+    "\n",
+    "For more details, please refer to the IDAES documentation: https://idaes-pse.readthedocs.io/en/stable"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setting up the problem in IDAES"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the following cell, we will be importing the necessary components from Pyomo and IDAES."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import objects from pyomo package \n",
+    "from pyomo.environ import ConcreteModel, SolverFactory, value\n",
+    "\n",
+    "# Import the main FlowsheetBlock from IDAES. The flowsheet block will contain the unit model\n",
+    "from idaes.core import FlowsheetBlock\n",
+    "\n",
+    "# Import the feed unit model\n",
+    "from idaes.models.unit_models import Feed\n",
+    "\n",
+    "# Import idaes logger to set output levels\n",
+    "import idaes.logger as idaeslog\n",
+    "\n",
+    "# Import the BTX_ideal property package to create a properties block for the flowsheet\n",
+    "from idaes.models.properties.activity_coeff_models import BTX_activity_coeff_VLE\n",
+    "\n",
+    "# Import the degrees_of_freedom function from the idaes.core.util.model_statistics package\n",
+    "# DOF = Number of Model Variables - Number of Model Constraints\n",
+    "from idaes.core.util.model_statistics import degrees_of_freedom\n",
+    "\n",
+    "# Create the ConcreteModel and the FlowsheetBlock objects, and attach the flowsheet block to it.\n",
+    "m = ConcreteModel()\n",
+    "\n",
+    "m.fs = FlowsheetBlock(dynamic=False) # dynamic or ss flowsheet needs to be specified here\n",
+    "\n",
+    "# Add properties parameter block to the flowsheet with specifications\n",
+    "m.fs.properties = BTX_activity_coeff_VLE.BTXParameterBlock(valid_phase='Liq',\n",
+    "                                                           activity_coeff_model=\"Ideal\")\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The initial degrees of freedom is: 3\n"
+     ]
+    }
+   ],
+   "source": [
```

Comment:
I think there needs to be a brief markdown box before each cell to give a little bit of discussion about the next step.